cmd/root.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // Regex's used to validate service and key names
 var (
-	validKeyFormat     = regexp.MustCompile(`^[A-Za-z0-9-_]+$`)
-	validServiceFormat = regexp.MustCompile(`^[A-Za-z0-9-_]+$`)
+	validKeyFormat         = regexp.MustCompile(`^[\w\.]+$`)
+	validServiceFormat     = regexp.MustCompile(`^[\w]+(\.[\w]+)*$`)
+	validServicePathFormat = regexp.MustCompile(`^[\w]+(\/[\w\.]+)*$`)
```

Comment:
does not support dashes and underscores

Reply:
```
^[\w\.-]+$
^[\w-]+(\.[\w-]+)*$
^[\w-]+(\/[\w\.-]+)*$
```

Reply:
@AndreyMarchuk Good catches, I updated the test cases and the regex both for SSM and for the CMD. I will put it as a fixup, so you can see the changes, once we are ready to merge I can squash them.

Reply:
One slight comment, I left some of the old functionality where a `/./this` path/key is allowed, which I believe should be fine and it leaves on the user's hands using a proper naming convention.

Reply:
Awesome, thanks! I like that we are doing a quick progress.

Reply:
This should be ready to review now.